apps/platform-app/src/app/[locale]/(auth)/(components)/LinkedInLoginButton/LinkedInLoginButton.tsx

```diff
@@ -0,0 +1,33 @@
+'use client'
+import { AppRoutes } from '@/utils/routes'
+import { Button } from '@gdh/ui-system'
+import { LinkedIn } from '@gdh/ui-system/icons'
+import { signIn } from 'next-auth/react'
+import { useState } from 'react'
+import styles from '../GithubLoginButton/GithubLoginButton.module.scss'
+
+interface LinkedInLoginButtonProps {
+  label?: string
+  role: 'SPECIALIST' | 'HUNTER'
```

Comment:
**nit:** We should have enum for that, eg: `enum Roles` to prevent string duplication

Reply:
I have created Roles enum inside of ```src\app\[locale]\(auth)\_models\User.model.ts``` because we can not use Role enum from prisma here